src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_BOUND = 100;
+    private Random randomNumber = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
Consider whether `ColorSupplier` should be a class-level variable. If you plan to use the same `ColorSupplier` instance across multiple method calls, it makes sense to keep it as a class-level variable. Otherwise, you may instantiate it locally within the method where it is used.